engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CallCacheHashingJobActorDataSpec.scala

```diff
@@ -9,84 +9,84 @@ import cromwell.engine.workflow.lifecycle.execution.callcaching.CallCacheHashing
 import org.scalatest.concurrent.Eventually
 import org.scalatest.flatspec.AnyFlatSpecLike
 import org.scalatest.matchers.should.Matchers
-import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatest.prop.{TableDrivenPropertyChecks, TableFor3}
 
 class CallCacheHashingJobActorDataSpec extends TestKitSuite with AnyFlatSpecLike with BackendSpec with Matchers with Eventually with TableDrivenPropertyChecks {
   behavior of "CallCacheReadingJobActorData"
 
-  val fileHash1 = HashResult(HashKey("key"), HashValue("value"))
-  val fileHash2 = HashResult(HashKey("key2"), HashValue("value2"))
-  val fileHash3 = HashResult(HashKey("key3"), HashValue("value3"))
-  val fileHashRequest1 = SingleFileHashRequest(null, fileHash1.hashKey, null, null)
-  val fileHashRequest2 = SingleFileHashRequest(null, fileHash2.hashKey, null, null)
-  val fileHashRequest3 = SingleFileHashRequest(null, fileHash3.hashKey, null, null)
+  val fileHash1: HashResult = HashResult(HashKey("key"), HashValue("value"))
```

Comment:
TOL: Do we really prefer the explicit val type in cases like this? I've been seeing PRs with drive-by changes like this and, in a lot of cases, I can see that it's a benefit. Here, it just looks like an unnecessary repetition of the type.

Reply:
IntelliJ's style checking currently wants type ascriptions on all public members. I agree that most of the time it's helpful but this particular case does feel kind of silly. I can fix this instance by removing the ascriptions and making all of these members `private` (which they really should have been anyway).